README.md

```diff
@@ -1,139 +1,167 @@
+  jaeger-agent-lbcnv                            1/1       Running     0          1m
+  jaeger-collector-86d57594d5-7lktp             1/1       Running     1          1m
+  jaeger-query-f96b97b75-bm2sw                  1/1       Running     1          1m
+  kiali-59fd54974c-4c9xz                        1/1       Running     0          1m
+  openshift-ansible-istio-installer-job-ddvdr   0/1       Completed   0          4m
+  prometheus-75b849445c-8n8rv                   1/1       Running     0          2m
+  ```
 
-## Launcher Flow Setup
+- Remember the URL of the Istio ingress gateway:
 
-If the Booster is installed through the Launcher and the Continuous Delivery flow, no additional steps are necessary.
+  ```bash
+  ISTIO_INGRESS=$(oc get route istio-ingressgateway -n istio-system -o jsonpath='{.spec.host}')
+  ```
+## Deploy the application
 
-Skip to the _Use Cases_ section.
+### Launcher flow
 
-## Local Source to Image Build (S2I)
+If you installed Fabric8 Launcher, you can deploy the example application using its Continuous Delivery flow.
+In such case, no manual deployment steps are necessary.
 
-### Prepare the Namespace
+Go to the _Verifying use cases_ section.
 
-Create a new namespace/project:
+### Manual build
+
+#### Prepare the namespace
+
+Move to your namespace/project:
+
+```bash
+oc login -u developer
+oc project myproject
 ```
-oc new-project <whatever valid project name you want>
+
+This is the project to which the `anyuid` and `privileged` permissions were added during Istio preparation.
+
+#### Build and deploy the application
+
+You can build and deploy the example application using either Fabric8 Maven plugin, or OpenShift templates and S2I.
+
+##### Using Fabric8 Maven plugin
+
+Build and deploy the services directly:
+
+```bash
+cd cute-name-service
+mvn clean fabric8:deploy -Popenshift
+cd ..
+
+cd greeting-service
+mvn clean fabric8:deploy -Popenshift
+cd ..
 ```
 
-### Build and Deploy the Application
+##### Using S2I
 
-#### With Source to Image build (S2I)
+Run the following commands to import and apply the provided OpenShift templates.
+OpenShift will then build and deploy the services:
 
-Run the following commands to apply and execute the OpenShift templates that will configure and deploy the applications:
 ```bash
-find . | grep openshiftio | grep application | xargs -n 1 oc apply -f
+oc apply -f ./cute-name-service/.openshiftio/application.yaml
+oc new-app --template=thorntail-istio-tracing-cute-name
 
-oc new-app --template=thorntail-istio-tracing-greeting-service -p SOURCE_REPOSITORY_URL=https://github.com/thorntail-examples/istio-tracing -p SOURCE_REPOSITORY_REF=master -p SOURCE_REPOSITORY_DIR=greeting-service
-oc new-app --template=thorntail-istio-tracing-cute-name-service -p SOURCE_REPOSITORY_URL=https://github.com/thorntail-examples/istio-tracing -p SOURCE_REPOSITORY_REF=master -p SOURCE_REPOSITORY_DIR=cute-name-service
+oc apply -f ./greeting-service/.openshiftio/application.yaml
```

Comment:
This may not work for S2i  well - I mean how would one point to a non-master branch or SOURCE_REPOSITORY_REF can be still overridden via a command line property ? (I used to test newer Thorntail version from the non-master branch)

Reply:
Well, not a bit deal for someone to manually update application.yaml files to update the branches, realistically, now that you've added the FMP support it would be an easier option anyway, but may be worth updating the S2i section to advise how to update  SOURCE_REPOSITORY_REF if needed.

Reply:
Hmm, yea, I removed the parameters because they defaults are now set correctly in the template. But they can still be passed to `oc new-app` as they used to be. Let me add the info to the README.